CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.20)
 project(bla)
 
-set (CMAKE_CXX_STANDARD 11)
+set (CMAKE_CXX_STANDARD 17)
```

Comment:
Could we also use 20?

Reply:
Let's stick to 17 as we do not need c++ 20 or 23 features at the moment. This way the project stays compatible with a wider range of compilers, making it more accessible.